[2022-07-06 19:29:50,966] {processor.py:153} INFO - Started process (PID=46417) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:29:50,967] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:29:50,967] {logging_mixin.py:115} INFO - [2022-07-06 19:29:50,967] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:29:50,999] {logging_mixin.py:115} INFO - [2022-07-06 19:29:50,998] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:29:50,999] {logging_mixin.py:115} INFO - [2022-07-06 19:29:50,999] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:29:51,000] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:29:51,020] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2022-07-06 19:30:29,306] {processor.py:153} INFO - Started process (PID=46821) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:30:29,310] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:30:29,311] {logging_mixin.py:115} INFO - [2022-07-06 19:30:29,311] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:30:29,349] {logging_mixin.py:115} INFO - [2022-07-06 19:30:29,349] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:30:29,350] {logging_mixin.py:115} INFO - [2022-07-06 19:30:29,350] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:30:29,350] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:30:29,373] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2022-07-06 19:31:05,151] {processor.py:153} INFO - Started process (PID=46961) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:31:05,154] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:31:05,154] {logging_mixin.py:115} INFO - [2022-07-06 19:31:05,154] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:31:05,175] {logging_mixin.py:115} INFO - [2022-07-06 19:31:05,174] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:31:05,175] {logging_mixin.py:115} INFO - [2022-07-06 19:31:05,175] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:31:05,175] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:31:05,187] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2022-07-06 19:31:38,452] {processor.py:153} INFO - Started process (PID=47532) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:31:38,456] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:31:38,457] {logging_mixin.py:115} INFO - [2022-07-06 19:31:38,457] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:31:38,488] {logging_mixin.py:115} INFO - [2022-07-06 19:31:38,487] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:31:38,489] {logging_mixin.py:115} INFO - [2022-07-06 19:31:38,488] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:31:38,489] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:31:38,503] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-07-06 19:32:12,977] {processor.py:153} INFO - Started process (PID=47636) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:32:12,980] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:32:12,980] {logging_mixin.py:115} INFO - [2022-07-06 19:32:12,980] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:32:13,006] {logging_mixin.py:115} INFO - [2022-07-06 19:32:13,006] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:32:13,006] {logging_mixin.py:115} INFO - [2022-07-06 19:32:13,006] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:32:13,006] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:32:13,018] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2022-07-06 19:32:48,973] {processor.py:153} INFO - Started process (PID=47720) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:32:48,975] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:32:48,976] {logging_mixin.py:115} INFO - [2022-07-06 19:32:48,976] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:32:48,994] {logging_mixin.py:115} INFO - [2022-07-06 19:32:48,994] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:32:48,995] {logging_mixin.py:115} INFO - [2022-07-06 19:32:48,995] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:32:48,995] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:32:49,007] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2022-07-06 19:33:25,070] {processor.py:153} INFO - Started process (PID=47920) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:33:25,078] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:33:25,079] {logging_mixin.py:115} INFO - [2022-07-06 19:33:25,079] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:33:25,114] {logging_mixin.py:115} INFO - [2022-07-06 19:33:25,113] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:33:25,114] {logging_mixin.py:115} INFO - [2022-07-06 19:33:25,114] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:33:25,115] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:33:25,142] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2022-07-06 19:34:00,050] {processor.py:153} INFO - Started process (PID=48042) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:34:00,051] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:34:00,051] {logging_mixin.py:115} INFO - [2022-07-06 19:34:00,051] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:34:00,069] {logging_mixin.py:115} INFO - [2022-07-06 19:34:00,068] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:34:00,069] {logging_mixin.py:115} INFO - [2022-07-06 19:34:00,069] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:34:00,069] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:34:00,086] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2022-07-06 19:34:33,734] {processor.py:153} INFO - Started process (PID=48183) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:34:33,737] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:34:33,738] {logging_mixin.py:115} INFO - [2022-07-06 19:34:33,737] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:34:33,759] {logging_mixin.py:115} INFO - [2022-07-06 19:34:33,759] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:34:33,760] {logging_mixin.py:115} INFO - [2022-07-06 19:34:33,760] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:34:33,760] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:34:33,773] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2022-07-06 19:35:08,459] {processor.py:153} INFO - Started process (PID=48286) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:35:08,461] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:35:08,462] {logging_mixin.py:115} INFO - [2022-07-06 19:35:08,462] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:35:08,480] {logging_mixin.py:115} INFO - [2022-07-06 19:35:08,479] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:35:08,480] {logging_mixin.py:115} INFO - [2022-07-06 19:35:08,480] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:35:08,480] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:35:08,492] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2022-07-06 19:35:42,426] {processor.py:153} INFO - Started process (PID=48395) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:35:42,427] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:35:42,428] {logging_mixin.py:115} INFO - [2022-07-06 19:35:42,427] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:35:42,447] {logging_mixin.py:115} INFO - [2022-07-06 19:35:42,446] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:35:42,447] {logging_mixin.py:115} INFO - [2022-07-06 19:35:42,447] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:35:42,447] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:35:42,460] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2022-07-06 19:36:15,490] {processor.py:153} INFO - Started process (PID=48538) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:36:15,493] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:36:15,494] {logging_mixin.py:115} INFO - [2022-07-06 19:36:15,494] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:36:15,521] {logging_mixin.py:115} INFO - [2022-07-06 19:36:15,520] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:36:15,521] {logging_mixin.py:115} INFO - [2022-07-06 19:36:15,521] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:36:15,521] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:36:15,534] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2022-07-06 19:36:49,545] {processor.py:153} INFO - Started process (PID=48606) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:36:49,548] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:36:49,548] {logging_mixin.py:115} INFO - [2022-07-06 19:36:49,548] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:36:49,572] {logging_mixin.py:115} INFO - [2022-07-06 19:36:49,571] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:36:49,572] {logging_mixin.py:115} INFO - [2022-07-06 19:36:49,572] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:36:49,572] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:36:49,586] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-07-06 19:37:24,176] {processor.py:153} INFO - Started process (PID=48928) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:37:24,177] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:37:24,178] {logging_mixin.py:115} INFO - [2022-07-06 19:37:24,177] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:37:24,219] {logging_mixin.py:115} INFO - [2022-07-06 19:37:24,219] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:37:24,220] {logging_mixin.py:115} INFO - [2022-07-06 19:37:24,220] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:37:24,220] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:37:24,237] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2022-07-06 19:37:59,738] {processor.py:153} INFO - Started process (PID=49089) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:37:59,741] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:37:59,742] {logging_mixin.py:115} INFO - [2022-07-06 19:37:59,741] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:37:59,759] {logging_mixin.py:115} INFO - [2022-07-06 19:37:59,759] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:37:59,759] {logging_mixin.py:115} INFO - [2022-07-06 19:37:59,759] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:37:59,759] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:37:59,771] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2022-07-06 19:38:37,942] {processor.py:153} INFO - Started process (PID=49318) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:38:37,959] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:38:37,959] {logging_mixin.py:115} INFO - [2022-07-06 19:38:37,959] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:38:38,011] {logging_mixin.py:115} INFO - [2022-07-06 19:38:38,009] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:38:38,012] {logging_mixin.py:115} INFO - [2022-07-06 19:38:38,012] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:38:38,014] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:38:38,034] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2022-07-06 19:39:13,202] {processor.py:153} INFO - Started process (PID=49675) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:39:13,206] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:39:13,207] {logging_mixin.py:115} INFO - [2022-07-06 19:39:13,207] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:39:13,241] {logging_mixin.py:115} INFO - [2022-07-06 19:39:13,240] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:39:13,241] {logging_mixin.py:115} INFO - [2022-07-06 19:39:13,241] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:39:13,242] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:39:13,260] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2022-07-06 19:39:48,362] {processor.py:153} INFO - Started process (PID=49822) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:39:48,365] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:39:48,365] {logging_mixin.py:115} INFO - [2022-07-06 19:39:48,365] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:39:48,385] {logging_mixin.py:115} INFO - [2022-07-06 19:39:48,384] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:39:48,385] {logging_mixin.py:115} INFO - [2022-07-06 19:39:48,385] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:39:48,385] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:39:48,397] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2022-07-06 19:41:13,479] {processor.py:153} INFO - Started process (PID=50954) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:41:13,482] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:41:13,482] {logging_mixin.py:115} INFO - [2022-07-06 19:41:13,482] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:41:13,501] {logging_mixin.py:115} INFO - [2022-07-06 19:41:13,501] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:41:13,502] {logging_mixin.py:115} INFO - [2022-07-06 19:41:13,502] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:41:13,502] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:41:13,517] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2022-07-06 19:41:49,163] {processor.py:153} INFO - Started process (PID=51238) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:41:49,166] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:41:49,167] {logging_mixin.py:115} INFO - [2022-07-06 19:41:49,167] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:41:49,193] {logging_mixin.py:115} INFO - [2022-07-06 19:41:49,192] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:41:49,193] {logging_mixin.py:115} INFO - [2022-07-06 19:41:49,193] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:41:49,193] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:41:49,206] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2022-07-06 19:42:23,338] {processor.py:153} INFO - Started process (PID=51349) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:42:23,340] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:42:23,341] {logging_mixin.py:115} INFO - [2022-07-06 19:42:23,341] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:42:23,398] {logging_mixin.py:115} INFO - [2022-07-06 19:42:23,397] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:42:23,399] {logging_mixin.py:115} INFO - [2022-07-06 19:42:23,399] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:42:23,400] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:42:23,429] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2022-07-06 19:42:59,350] {processor.py:153} INFO - Started process (PID=51450) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:42:59,351] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:42:59,352] {logging_mixin.py:115} INFO - [2022-07-06 19:42:59,352] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:42:59,371] {logging_mixin.py:115} INFO - [2022-07-06 19:42:59,371] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:42:59,371] {logging_mixin.py:115} INFO - [2022-07-06 19:42:59,371] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:42:59,372] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:42:59,383] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2022-07-06 19:43:35,838] {processor.py:153} INFO - Started process (PID=51554) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:43:35,841] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:43:35,842] {logging_mixin.py:115} INFO - [2022-07-06 19:43:35,842] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:43:35,873] {logging_mixin.py:115} INFO - [2022-07-06 19:43:35,873] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:43:35,874] {logging_mixin.py:115} INFO - [2022-07-06 19:43:35,874] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:43:35,874] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:43:35,892] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2022-07-06 19:44:13,370] {processor.py:153} INFO - Started process (PID=51654) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:44:13,372] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:44:13,373] {logging_mixin.py:115} INFO - [2022-07-06 19:44:13,373] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:44:13,395] {logging_mixin.py:115} INFO - [2022-07-06 19:44:13,394] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:44:13,395] {logging_mixin.py:115} INFO - [2022-07-06 19:44:13,395] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:44:13,395] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:44:13,410] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2022-07-06 19:44:48,945] {processor.py:153} INFO - Started process (PID=51811) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:44:48,948] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:44:48,948] {logging_mixin.py:115} INFO - [2022-07-06 19:44:48,948] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:44:48,969] {logging_mixin.py:115} INFO - [2022-07-06 19:44:48,968] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:44:48,970] {logging_mixin.py:115} INFO - [2022-07-06 19:44:48,970] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:44:48,970] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:44:48,982] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2022-07-06 19:45:23,019] {processor.py:153} INFO - Started process (PID=51927) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:45:23,019] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:45:23,020] {logging_mixin.py:115} INFO - [2022-07-06 19:45:23,020] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:45:23,039] {logging_mixin.py:115} INFO - [2022-07-06 19:45:23,038] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:45:23,039] {logging_mixin.py:115} INFO - [2022-07-06 19:45:23,039] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:45:23,039] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:45:23,051] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2022-07-06 19:46:03,320] {processor.py:153} INFO - Started process (PID=52053) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:46:03,321] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:46:03,321] {logging_mixin.py:115} INFO - [2022-07-06 19:46:03,321] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:46:03,351] {logging_mixin.py:115} INFO - [2022-07-06 19:46:03,350] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:46:03,352] {logging_mixin.py:115} INFO - [2022-07-06 19:46:03,351] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:46:03,352] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:46:03,369] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-07-06 19:46:40,119] {processor.py:153} INFO - Started process (PID=52125) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:46:40,122] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:46:40,124] {logging_mixin.py:115} INFO - [2022-07-06 19:46:40,123] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:46:40,164] {logging_mixin.py:115} INFO - [2022-07-06 19:46:40,162] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:46:40,165] {logging_mixin.py:115} INFO - [2022-07-06 19:46:40,165] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:46:40,165] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:46:40,188] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2022-07-06 19:47:13,887] {processor.py:153} INFO - Started process (PID=52192) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:47:13,890] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:47:13,890] {logging_mixin.py:115} INFO - [2022-07-06 19:47:13,890] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:47:13,909] {logging_mixin.py:115} INFO - [2022-07-06 19:47:13,908] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:47:13,909] {logging_mixin.py:115} INFO - [2022-07-06 19:47:13,909] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:47:13,909] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:47:13,921] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2022-07-06 19:47:49,243] {processor.py:153} INFO - Started process (PID=52283) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:47:49,253] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:47:49,254] {logging_mixin.py:115} INFO - [2022-07-06 19:47:49,253] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:47:49,288] {logging_mixin.py:115} INFO - [2022-07-06 19:47:49,287] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:47:49,288] {logging_mixin.py:115} INFO - [2022-07-06 19:47:49,288] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:47:49,289] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:47:49,317] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2022-07-06 19:48:26,949] {processor.py:153} INFO - Started process (PID=52363) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:48:26,950] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:48:26,951] {logging_mixin.py:115} INFO - [2022-07-06 19:48:26,951] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:48:26,970] {logging_mixin.py:115} INFO - [2022-07-06 19:48:26,970] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:48:26,971] {logging_mixin.py:115} INFO - [2022-07-06 19:48:26,971] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:48:26,971] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:48:26,984] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2022-07-06 19:49:03,205] {processor.py:153} INFO - Started process (PID=52463) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:49:03,208] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:49:03,209] {logging_mixin.py:115} INFO - [2022-07-06 19:49:03,209] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:49:03,228] {logging_mixin.py:115} INFO - [2022-07-06 19:49:03,228] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:49:03,229] {logging_mixin.py:115} INFO - [2022-07-06 19:49:03,229] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:49:03,229] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:49:03,240] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2022-07-06 19:49:39,626] {processor.py:153} INFO - Started process (PID=52670) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:49:39,630] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:49:39,632] {logging_mixin.py:115} INFO - [2022-07-06 19:49:39,632] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:49:39,680] {logging_mixin.py:115} INFO - [2022-07-06 19:49:39,679] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:49:39,680] {logging_mixin.py:115} INFO - [2022-07-06 19:49:39,680] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:49:39,681] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:49:39,712] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.094 seconds
[2022-07-06 19:50:19,387] {processor.py:153} INFO - Started process (PID=52765) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:50:19,389] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:50:19,390] {logging_mixin.py:115} INFO - [2022-07-06 19:50:19,390] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:50:19,411] {logging_mixin.py:115} INFO - [2022-07-06 19:50:19,411] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:50:19,411] {logging_mixin.py:115} INFO - [2022-07-06 19:50:19,411] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:50:19,412] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:50:19,424] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2022-07-06 19:50:56,777] {processor.py:153} INFO - Started process (PID=52883) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:50:56,782] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:50:56,783] {logging_mixin.py:115} INFO - [2022-07-06 19:50:56,783] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:50:56,897] {logging_mixin.py:115} INFO - [2022-07-06 19:50:56,896] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:50:56,897] {logging_mixin.py:115} INFO - [2022-07-06 19:50:56,897] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:50:56,897] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:50:56,911] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.140 seconds
[2022-07-06 19:51:30,896] {processor.py:153} INFO - Started process (PID=52958) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:51:30,898] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:51:30,899] {logging_mixin.py:115} INFO - [2022-07-06 19:51:30,899] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:51:30,923] {logging_mixin.py:115} INFO - [2022-07-06 19:51:30,922] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:51:30,923] {logging_mixin.py:115} INFO - [2022-07-06 19:51:30,923] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:51:30,923] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:51:30,936] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-07-06 19:52:06,064] {processor.py:153} INFO - Started process (PID=53030) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:52:06,067] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:52:06,068] {logging_mixin.py:115} INFO - [2022-07-06 19:52:06,068] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:52:06,095] {logging_mixin.py:115} INFO - [2022-07-06 19:52:06,094] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:52:06,095] {logging_mixin.py:115} INFO - [2022-07-06 19:52:06,095] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:52:06,095] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:52:06,112] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-07-06 19:52:41,769] {processor.py:153} INFO - Started process (PID=53104) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:52:41,772] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:52:41,773] {logging_mixin.py:115} INFO - [2022-07-06 19:52:41,773] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:52:41,792] {logging_mixin.py:115} INFO - [2022-07-06 19:52:41,791] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:52:41,792] {logging_mixin.py:115} INFO - [2022-07-06 19:52:41,792] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:52:41,792] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:52:41,804] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2022-07-06 19:53:18,327] {processor.py:153} INFO - Started process (PID=53197) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:53:18,328] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:53:18,328] {logging_mixin.py:115} INFO - [2022-07-06 19:53:18,328] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:53:18,349] {logging_mixin.py:115} INFO - [2022-07-06 19:53:18,349] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:53:18,349] {logging_mixin.py:115} INFO - [2022-07-06 19:53:18,349] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:53:18,350] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:53:18,363] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2022-07-06 19:53:58,565] {processor.py:153} INFO - Started process (PID=53311) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:53:58,568] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:53:58,569] {logging_mixin.py:115} INFO - [2022-07-06 19:53:58,569] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:53:58,600] {logging_mixin.py:115} INFO - [2022-07-06 19:53:58,599] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:53:58,600] {logging_mixin.py:115} INFO - [2022-07-06 19:53:58,600] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:53:58,600] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:53:58,619] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2022-07-06 19:54:36,766] {processor.py:153} INFO - Started process (PID=53459) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:54:36,769] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:54:36,769] {logging_mixin.py:115} INFO - [2022-07-06 19:54:36,769] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:54:36,791] {logging_mixin.py:115} INFO - [2022-07-06 19:54:36,790] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:54:36,791] {logging_mixin.py:115} INFO - [2022-07-06 19:54:36,791] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:54:36,791] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:54:36,803] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2022-07-06 19:55:11,410] {processor.py:153} INFO - Started process (PID=53559) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:55:11,411] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:55:11,412] {logging_mixin.py:115} INFO - [2022-07-06 19:55:11,412] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:55:11,501] {logging_mixin.py:115} INFO - [2022-07-06 19:55:11,500] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:55:11,502] {logging_mixin.py:115} INFO - [2022-07-06 19:55:11,502] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:55:11,503] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:55:11,533] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.146 seconds
[2022-07-06 19:55:46,679] {processor.py:153} INFO - Started process (PID=53654) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:55:46,682] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:55:46,683] {logging_mixin.py:115} INFO - [2022-07-06 19:55:46,682] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:55:46,723] {logging_mixin.py:115} INFO - [2022-07-06 19:55:46,715] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:55:46,723] {logging_mixin.py:115} INFO - [2022-07-06 19:55:46,723] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:55:46,724] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:55:46,743] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2022-07-06 19:56:24,081] {processor.py:153} INFO - Started process (PID=53725) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:56:24,082] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:56:24,083] {logging_mixin.py:115} INFO - [2022-07-06 19:56:24,083] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:56:24,111] {logging_mixin.py:115} INFO - [2022-07-06 19:56:24,111] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:56:24,111] {logging_mixin.py:115} INFO - [2022-07-06 19:56:24,111] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:56:24,112] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:56:24,124] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2022-07-06 19:57:00,278] {processor.py:153} INFO - Started process (PID=53797) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:57:00,280] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:57:00,281] {logging_mixin.py:115} INFO - [2022-07-06 19:57:00,281] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:57:00,300] {logging_mixin.py:115} INFO - [2022-07-06 19:57:00,300] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:57:00,300] {logging_mixin.py:115} INFO - [2022-07-06 19:57:00,300] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:57:00,301] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:57:00,312] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2022-07-06 19:57:35,262] {processor.py:153} INFO - Started process (PID=53871) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:57:35,267] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:57:35,269] {logging_mixin.py:115} INFO - [2022-07-06 19:57:35,268] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:57:35,305] {logging_mixin.py:115} INFO - [2022-07-06 19:57:35,303] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:57:35,306] {logging_mixin.py:115} INFO - [2022-07-06 19:57:35,305] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:57:35,306] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:57:35,328] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2022-07-06 19:58:12,170] {processor.py:153} INFO - Started process (PID=53973) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:58:12,173] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:58:12,174] {logging_mixin.py:115} INFO - [2022-07-06 19:58:12,174] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:58:12,197] {logging_mixin.py:115} INFO - [2022-07-06 19:58:12,196] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:58:12,197] {logging_mixin.py:115} INFO - [2022-07-06 19:58:12,197] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:58:12,197] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:58:12,210] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2022-07-06 19:58:47,753] {processor.py:153} INFO - Started process (PID=54067) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:58:47,757] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:58:47,759] {logging_mixin.py:115} INFO - [2022-07-06 19:58:47,758] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:58:47,784] {logging_mixin.py:115} INFO - [2022-07-06 19:58:47,783] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:58:47,784] {logging_mixin.py:115} INFO - [2022-07-06 19:58:47,784] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:58:47,784] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:58:47,802] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-07-06 19:59:23,299] {processor.py:153} INFO - Started process (PID=54167) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:59:23,300] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:59:23,301] {logging_mixin.py:115} INFO - [2022-07-06 19:59:23,301] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:59:23,320] {logging_mixin.py:115} INFO - [2022-07-06 19:59:23,319] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:59:23,320] {logging_mixin.py:115} INFO - [2022-07-06 19:59:23,320] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:59:23,320] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:59:23,333] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2022-07-06 19:59:57,382] {processor.py:153} INFO - Started process (PID=54290) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:59:57,382] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 19:59:57,383] {logging_mixin.py:115} INFO - [2022-07-06 19:59:57,383] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:59:57,405] {logging_mixin.py:115} INFO - [2022-07-06 19:59:57,405] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 19:59:57,406] {logging_mixin.py:115} INFO - [2022-07-06 19:59:57,406] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 19:59:57,406] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 19:59:57,418] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2022-07-06 20:00:31,754] {processor.py:153} INFO - Started process (PID=54402) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:00:31,757] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:00:31,758] {logging_mixin.py:115} INFO - [2022-07-06 20:00:31,758] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:00:31,777] {logging_mixin.py:115} INFO - [2022-07-06 20:00:31,776] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:00:31,777] {logging_mixin.py:115} INFO - [2022-07-06 20:00:31,777] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:00:31,777] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:00:31,789] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2022-07-06 20:01:08,357] {processor.py:153} INFO - Started process (PID=54548) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:01:08,360] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:01:08,361] {logging_mixin.py:115} INFO - [2022-07-06 20:01:08,361] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:01:08,382] {logging_mixin.py:115} INFO - [2022-07-06 20:01:08,382] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:01:08,383] {logging_mixin.py:115} INFO - [2022-07-06 20:01:08,383] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:01:08,383] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:01:08,396] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2022-07-06 20:01:43,616] {processor.py:153} INFO - Started process (PID=54639) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:01:43,619] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:01:43,620] {logging_mixin.py:115} INFO - [2022-07-06 20:01:43,620] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:01:43,642] {logging_mixin.py:115} INFO - [2022-07-06 20:01:43,642] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:01:43,642] {logging_mixin.py:115} INFO - [2022-07-06 20:01:43,642] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:01:43,643] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:01:43,658] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-07-06 20:02:20,321] {processor.py:153} INFO - Started process (PID=54827) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:02:20,323] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:02:20,324] {logging_mixin.py:115} INFO - [2022-07-06 20:02:20,324] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:02:20,346] {logging_mixin.py:115} INFO - [2022-07-06 20:02:20,345] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:02:20,346] {logging_mixin.py:115} INFO - [2022-07-06 20:02:20,346] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:02:20,346] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:02:20,360] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2022-07-06 20:02:55,647] {processor.py:153} INFO - Started process (PID=54903) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:02:55,651] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:02:55,653] {logging_mixin.py:115} INFO - [2022-07-06 20:02:55,652] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:02:55,675] {logging_mixin.py:115} INFO - [2022-07-06 20:02:55,675] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:02:55,676] {logging_mixin.py:115} INFO - [2022-07-06 20:02:55,676] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:02:55,676] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:02:55,691] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2022-07-06 20:03:34,034] {processor.py:153} INFO - Started process (PID=54985) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:03:34,035] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:03:34,036] {logging_mixin.py:115} INFO - [2022-07-06 20:03:34,036] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:03:34,078] {logging_mixin.py:115} INFO - [2022-07-06 20:03:34,078] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:03:34,078] {logging_mixin.py:115} INFO - [2022-07-06 20:03:34,078] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:03:34,079] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:03:34,095] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2022-07-06 20:04:13,284] {processor.py:153} INFO - Started process (PID=55133) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:04:13,287] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:04:13,287] {logging_mixin.py:115} INFO - [2022-07-06 20:04:13,287] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:04:13,308] {logging_mixin.py:115} INFO - [2022-07-06 20:04:13,307] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:04:13,308] {logging_mixin.py:115} INFO - [2022-07-06 20:04:13,308] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:04:13,308] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:04:13,321] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2022-07-06 20:04:46,353] {processor.py:153} INFO - Started process (PID=55388) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:04:46,354] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:04:46,354] {logging_mixin.py:115} INFO - [2022-07-06 20:04:46,354] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:04:46,375] {logging_mixin.py:115} INFO - [2022-07-06 20:04:46,375] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:04:46,376] {logging_mixin.py:115} INFO - [2022-07-06 20:04:46,376] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:04:46,376] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:04:46,388] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2022-07-06 20:05:22,273] {processor.py:153} INFO - Started process (PID=55473) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:05:22,276] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:05:22,277] {logging_mixin.py:115} INFO - [2022-07-06 20:05:22,277] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:05:22,299] {logging_mixin.py:115} INFO - [2022-07-06 20:05:22,298] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:05:22,299] {logging_mixin.py:115} INFO - [2022-07-06 20:05:22,299] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:05:22,300] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:05:22,315] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-07-06 20:05:58,320] {processor.py:153} INFO - Started process (PID=55549) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:05:58,321] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:05:58,321] {logging_mixin.py:115} INFO - [2022-07-06 20:05:58,321] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:05:58,345] {logging_mixin.py:115} INFO - [2022-07-06 20:05:58,345] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:05:58,346] {logging_mixin.py:115} INFO - [2022-07-06 20:05:58,346] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:05:58,346] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:05:58,359] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2022-07-06 20:06:35,612] {processor.py:153} INFO - Started process (PID=55641) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:06:35,615] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:06:35,615] {logging_mixin.py:115} INFO - [2022-07-06 20:06:35,615] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:06:35,641] {logging_mixin.py:115} INFO - [2022-07-06 20:06:35,641] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:06:35,641] {logging_mixin.py:115} INFO - [2022-07-06 20:06:35,641] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:06:35,642] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:06:35,657] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2022-07-06 20:07:12,226] {processor.py:153} INFO - Started process (PID=55719) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:07:12,229] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:07:12,230] {logging_mixin.py:115} INFO - [2022-07-06 20:07:12,230] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:07:12,254] {logging_mixin.py:115} INFO - [2022-07-06 20:07:12,254] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:07:12,255] {logging_mixin.py:115} INFO - [2022-07-06 20:07:12,254] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:07:12,255] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:07:12,276] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2022-07-06 20:07:49,326] {processor.py:153} INFO - Started process (PID=55807) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:07:49,329] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:07:49,330] {logging_mixin.py:115} INFO - [2022-07-06 20:07:49,330] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:07:49,354] {logging_mixin.py:115} INFO - [2022-07-06 20:07:49,353] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:07:49,354] {logging_mixin.py:115} INFO - [2022-07-06 20:07:49,354] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:07:49,354] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:07:49,367] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-07-06 20:08:22,671] {processor.py:153} INFO - Started process (PID=55880) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:08:22,674] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:08:22,675] {logging_mixin.py:115} INFO - [2022-07-06 20:08:22,675] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:08:22,696] {logging_mixin.py:115} INFO - [2022-07-06 20:08:22,696] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:08:22,697] {logging_mixin.py:115} INFO - [2022-07-06 20:08:22,696] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:08:22,697] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:08:22,716] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2022-07-06 20:09:00,310] {processor.py:153} INFO - Started process (PID=56043) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:09:00,312] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:09:00,313] {logging_mixin.py:115} INFO - [2022-07-06 20:09:00,313] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:09:00,334] {logging_mixin.py:115} INFO - [2022-07-06 20:09:00,333] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:09:00,334] {logging_mixin.py:115} INFO - [2022-07-06 20:09:00,334] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:09:00,334] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:09:00,347] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2022-07-06 20:09:39,963] {processor.py:153} INFO - Started process (PID=56401) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:09:39,966] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:09:39,967] {logging_mixin.py:115} INFO - [2022-07-06 20:09:39,967] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:09:39,994] {logging_mixin.py:115} INFO - [2022-07-06 20:09:39,994] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:09:39,995] {logging_mixin.py:115} INFO - [2022-07-06 20:09:39,995] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:09:39,995] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:09:40,013] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-07-06 20:10:18,200] {processor.py:153} INFO - Started process (PID=56863) to work on /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:10:18,203] {processor.py:641} INFO - Processing file /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 20:10:18,204] {logging_mixin.py:115} INFO - [2022-07-06 20:10:18,204] {dagbag.py:507} INFO - Filling up the DagBag from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:10:18,231] {logging_mixin.py:115} INFO - [2022-07-06 20:10:18,230] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 20:10:18,231] {logging_mixin.py:115} INFO - [2022-07-06 20:10:18,231] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 20:10:18,232] {processor.py:653} WARNING - No viable dags retrieved from /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 20:10:18,248] {processor.py:161} INFO - Processing /home/hat/dev-env/10Acadamy/week_9/Speech_to_text_data_collector/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
